apps/vr-tests/src/stories/Badge.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { Badge, BadgeCommons } from '@fluentui/react-badge';
+import { Circle12Regular } from '@fluentui/react-icons';
+import { mergeClasses, makeStyles } from '@fluentui/react-make-styles';
+
+const badgeColors: BadgeCommons['color'][] = [
+  'brand',
+  'danger',
+  'important',
+  'informative',
+  'severe',
+  'subtle',
+  'success',
+  'warning',
+];
+
+const badgeAppearances: BadgeCommons['appearance'][] = ['filled', 'outline', 'tint', 'ghost'];
+
+const useStyles = makeStyles({
+  container: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+
+  badgeContainer: {
+    display: 'flex',
+    alignItems: 'center',
+    gap: '5px',
+    padding: '5px',
+  },
+
+  label: {
+    marginLeft: '10px',
+  },
+
+  brandContainer: theme => ({
+    backgroundColor: theme.colorBrandBackgroundStatic,
+  }),
+});
+
+const BadgeAppearanceTemplate: React.FC<{ appearance: BadgeCommons['appearance'] }> = ({
+  appearance,
+}) => {
+  const styles = useStyles();
+
+  const badges: Record<BadgeCommons['color'], JSX.Element[]> = {
+    brand: [],
```

Comment:
nit: it would be great to match the order in the design spec for quick manual check.